website/content/en/docs/olm-integration/generating-a-csv.md

```diff
@@ -1,208 +1,248 @@
+[above](#kustomize-files) to idempotently regenerate a CSV.
 
-  By default, the command starts an interactive prompt if some base CSV is not present,
-  either in the destination directory or a package manifests directory named with version `--from-version=<version>`.
+The following resource kinds are typically included in a CSV, which are addressed by `config/manifests/bases/kustomization.yaml`:
+    - `Role`: define Operator permissions within a namespace.
+    - `ClusterRole`: define cluster-wide Operator permissions.
+    - `Deployment`: define how the Operator's operand is run in pods.
+    - `CustomResourceDefinition` (CRD): definitions of custom objects your Operator reconciles.
+    - Custom resource examples: examples of objects adhering to the spec of a particular CRD.
 
-### Output
+## Generate your first release
 
-By default `generate csv` will create a CSV and copy CRDs in the [*bundle*][doc-bundle]
-manifests format:
+You've recently run `operator-sdk init` and created your APIs with `operator-sdk create api`. Now you'd like to
+package your Operator for deployment by OLM. Your Operator is at version `v0.0.1`; the `Makefile` variable `VERSION`
+should be set to `0.0.1`.
 
-```
-deploy/
-└── olm-catalog
-    └── <operator-name>
-        └── manifests
-            ├── cache.example.com_memcacheds_crd.yaml
-            └── <operator-name>.clusterserviceversion.yaml
-```
+### Bundle format
 
-Setting `--make-manifests=false` will create a CSV and package manifest in the
-[package manifests format](#package-manifests-format):
+A [bundle][bundle] consists of manifests (CSV and CRDs) and metadata that define an Operator
+at a particular version. You may have also heard of a bundle image. From the bundle docs:
 
-```
-deploy/
-└── olm-catalog
-    └── <operator-name>
-        ├── 0.0.1
-        │   └── <operator-name>.v0.0.1.clusterserviceversion.yaml
-        └── <operator-name>.package.yaml
-```
+> An Operator Bundle is built as a scratch (non-runnable) container image that
+> contains operator manifests and specific metadata in designated directories
+> inside the image. Then, it can be pushed and pulled from an OCI-compliant
+> container registry. Ultimately, an operator bundle will be used by Operator
+> Registry and OLM to install an operator in OLM-enabled clusters.
 
-Set the `--ouput-dir` flag to change where the CSV is generated.
+At this stage in your Operator's development, we only need to worry about generating bundle files;
+bundle images become important once you're ready to [publish][operatorhub] your Operator.
 
-**Note:** `generate csv` will populate many but not all fields in your CSV
-automatically. Subsequent calls to `generate csv` will warn you of missing
-required fields. See the list of fields [below](#csv-fields) for more information.
+Bundle metadata contains information about a particular Operator version available in a registry.
```

Comment:
```suggestion
Bundle metadata contains information about a particular Operator version.
```
It may be confusing since we have not yet talked about bundle image or operator registry.

Reply:
I've added a link instead since this information is important.